config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_sqlinstances.sql.cnrm.cloud.google.com.yaml

```diff
@@ -84,14 +85,14 @@ spec:
                   - external
                 properties:
                   external:
-                    description: 'Allowed value: The `selfLink` field of a `KMSCryptoKey`
-                      resource.'
+                    description: The KMSCryptoKeyRef selfLink, when not managed by
+                      KCC.
```

Comment:
`KCC` is not a valid brand name. If we want to mention the product officially in public (CRD is one of the places like this), we should use `Config Connector`.

Reply:
fixed